tests/components/switch/test_init.py

```diff
@@ -83,15 +90,22 @@ def test_setup_two_platforms(self):
             self.hass,
             switch.DOMAIN,
             {
-                switch.DOMAIN: {CONF_PLATFORM: "test"},
-                "{} 2".format(switch.DOMAIN): {CONF_PLATFORM: "test2"},
+                switch.DOMAIN: {
+                    CONF_PLATFORM: "test"
+                },
+                "{} 2".format(switch.DOMAIN): {
+                    CONF_PLATFORM: "test2"
+                },
             },
         )
 
 
 async def test_switch_context(hass, hass_admin_user):
     """Test that switch context works."""
-    assert await async_setup_component(hass, "switch", {"switch": {"platform": "test"}})
+    assert await async_setup_component(hass, "switch",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)